### ISSUE TITLE
OneDSInspector tool removed from the repo

### ISSUE THREAD

**Issue by lalitb** [microsoft/cpp_client_telemetry#1274]
The tool is removed from the repo temporarily (ref #1273) as there were some build errors, and it was not part of the CI pipelines. The tool is used by various teams for debugging 1DS traffic through fiddler. This issue is created for tracking purpose to fix the build, and bring it back.

**Comment by lalitb** [comment]
Other option - https://github.com/microsoft/cpp_client_telemetry/pull/336
Also related issue - https://github.com/microsoft/cpp_client_telemetry/issues/652

**Comment by maxgolov** [comment]
Yes, our team ( Microsoft Mesh ) uses the decoder, as well as the C# test server in our test automation. We need it back.

**Comment by maxgolov** [comment]
@ThomsonTan - you can assign it to me. I would be able to handle it in 2024 Q3. Meanwhile I'd recommend our team to use an older snapshot. I need to integrate the tooling in some of our internal quality monitoring tooling.
